dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -6,11 +6,11 @@ get_cmd_version() {
     fi
 
     local cmd="$1"
-    if command -v "$cmd" 2>&1 >/dev/null; then
-        ver=$("$cmd" --version 2> /dev/null | head -n 1)
+    if command -v $cmd 2>&1 >/dev/null; then
```

Comment:
How about adding optional extra arguments to this function instead, which can be used to pass arguments?
```bash
get_cmd_version() {
    if [ -z "$1" ]; then
        return
    fi

    local cmd="$1"
    shift 1
    local args=--version
    if [ $# -gt 0 ]; then
        args="$@"
    fi
    if command -v $cmd 2>&1 >/dev/null; then
        ver=$($cmd $args 2> /dev/null | head -n 1)
        # some tools (eg. openocd) print version info to stderr
        if [ -z "$ver" ]; then
            ver=$($cmd $args 2>&1 | head -n 1)
        fi
        if [ -z "$ver" ]; then
            ver="error"
        fi
    else
        ver="missing"
    fi

    printf "%s" "$ver"
}
```
and
```sh
printf "%23s: %s\n" "flake8" "$(get_cmd_version python3 -m flake8 --version)"
```

Though, on failure we get this error message instead (because of the captured stderr output):
```
flake8: /usr/lib/python-exec/python3.6/python3: No module named fake8
```